examples/test_fixtures/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_version = "~> 0.10.6"
+}
+
+provider "aws" {
+  region  = "${var.aws_region}"
+  version = "1.0.0"
+}
+
+provider "template" {
+  version = "1.0.0"
+}
+
+resource "aws_iam_server_certificate" "fixture_cert" {
+  name             = "test_cert-${data.aws_caller_identity.fixtures.account_id}"
+  certificate_body = "${file("${path.module}/../../../examples/test_fixtures/certs/example.crt.pem")}"
+  private_key      = "${file("${path.module}/../../../examples/test_fixtures/certs/example.key.pem")}"
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+module "vpc" {
```

Comment:
Can you rely on default VPC which is already created? I use it in other examples and I think it will work for your example too.

Reply:
I typically delete the default VPC and I've known others and other organizations to do the same (security concern, not controlled by automation). I'd prefer to bring with my module an example that creates everything it needs. That way it's self sufficient.

I'll be migrating this to the registry VPC module instead.